eZ/Bundle/EzPublishCoreBundle/Command/ReindexCommand.php

```diff
@@ -68,6 +68,6 @@ protected function execute(InputInterface $input, OutputInterface $output)
             throw new RuntimeException("'--iteration-count' option should be > 0, got '{$iterationCount}'");
         }
 
-        $this->searchIndexer->createSearchIndex($output, intval($iterationCount), empty($noCommit));
+        $this->searchIndexer->createSearchIndex($output, (int) $iterationCount, empty($noCommit));
```

Comment:
This is `modernize_types_casting`. It applies only to code introduced by me, so it seems I was the only fan of `intval`. With the direction PHP7 is going to, now it makes more sense to apply normal casting.